src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas2/StatusUpdateService.kt

```diff
@@ -95,7 +95,7 @@ class StatusUpdateService(
           eventType = EventType.applicationStatusUpdated,
           eventDetails = Cas2ApplicationStatusUpdatedEventDetails(
             applicationId = application.id,
-            applicationUrl = applicationUrlTemplate.replace("id", application.id.toString()),
+            applicationUrl = applicationUrlTemplate.replace("#id", application.id.toString()),
```

Comment:
@pezholio thanks Stu, your suggestion worked well and nudged me towards verifying that the whole URL is correct -- which allowed me to pick up a further bug! see above line 98